src/main/java/seedu/address/model/expenses/ExpenseType.java

```diff
@@ -11,9 +11,8 @@
  * All products and services have a unique menu number.
  */
 public class ExpenseType {
-
-    public static final Map<String, String> numberToName;
-    public static final Map<String, Double> numberToPrice;
+    private static final Map<String, String> numberToName;
+    private static final Map<String, Double> numberToPrice;
     static {
         Map<String, String> numberToNameDummy = new HashMap<>();
```

Comment:
Okay I understand this is a dummy example of our menu catalog. Next step is serializing these in a text format (e.g. XML). However, I read that anything `static` is not serializable. You may need to use other ways, such as loading manually from a text file and doing `hashmap.put()` for each item.

Also, this is just my opinion, but I think having 1 Hashmap<String, ExpenseType> would be better than having 2 Hashmaps - <String, Name> and <String, Price>.